### PR TITLE
Update to latest api explorer, remove unused files

### PR DESCRIPTION
**Pull request by mlee-asana**
Gets latest changes from https://github.com/Asana/api-explorer/pull/89

### CODE REVIEW

File: source/explorer/popup_receiver.html [hunk 1]
```diff
@@ -1,17 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
-<html>
-<head>
-    <title>Asana Oauth Example: Browser Popup Receiver</title>
-    <script src="asana-explorer.js"></script>
```
Comment (mlee-asana) [inline source/explorer/popup_receiver.html:5]: this wouldn't work, `asana-explorer.js` isn't in this directory - could probably just remove this file?

Comment (jv-asana) [reply source/explorer/popup_receiver.html:5]: Hmmm. Good point the `index.html` file actually looks in the right directory for it: https://github.com/Asana/developer-docs/blob/master/source/explorer/index.html#L5

Comment (mlee-asana) [reply source/explorer/popup_receiver.html:5]: I'm pretty sure auth is just done via JS within the same window so this is likely unused

File: source/explorer/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
     <title>Asana API Explorer</title>
-    <script src="../javascripts/explorer/asana-explorer.js"></script>
+    <script src="../javascripts/explorer/asana-explorer-min.js"></script>
```
Comment (jv-asana) [inline source/explorer/index.html:5]: True. We should have done this earlier 😜

File: source/explorer/popup_receiver.html (hunk 1 above)
Comment (jv-asana) [reply source/explorer/popup_receiver.html:5]: If this file is being removed should the `REDIRECT_URI ` be updated as well? https://github.com/Asana/api-explorer/blob/6033dd82f04d8f99ddb3225f7694a1a9042d8f60/src/constants.ts#L9-L31

Comment (jv-asana) [reply source/explorer/popup_receiver.html:5]: Looks like we are passing that `REDIRECT_URI` into the client https://github.com/Asana/api-explorer/blob/6033dd82f04d8f99ddb3225f7694a1a9042d8f60/src/components/explorer.ts#L35. I am guessing it is redirecting to `https://asana.github.io/developer-docs/explorer/popup_receiver.html` (`https://developers.asana.com/explorer/popup_receiver.html`) after the auth. The current behavior takes you to `https://developers.asana.com/explorer`.

**Review by jv-asana — APPROVED** [review]
I think this should be ok. If this breaks we can revert.

**Comment by jv-asana** [comment]
Giving it some thought. I think the redirect URI is being set in the app settings itself.

File: source/explorer/popup_receiver.html (hunk 1 above)
Comment (mlee-asana) [reply source/explorer/popup_receiver.html:5]: yeah, took another look and it seems like it's supposed to redirect there? but with the way the script is referenced it should error out. yeah i'll deploy it out and test it that it works

**Pull request merged**